.buildkite/test-pipeline.yaml

```diff
@@ -369,7 +369,7 @@ steps:
     - pytest -v -s models/encoder_decoder/language -m core_model
     - pytest -v -s models/encoder_decoder/vision_language -m core_model
 
-- label: Multi-Modal Models Test (Extended) # 1h15m
+- label: Multi-Modal Models Test (Extended 1) # 1h15m
```

Comment:
Maybe we can rename this `Extended Multi-Modal Models Test 1` to be consistent with LoRA Test 1 or Kernels Test 1?